PyNuclei/PyNuclei.py

```diff
@@ -484,50 +467,48 @@ def scan(self, host, templates=list(), userAgent="", rateLimit=150, verbose=Fals
 		if self.stopAfter:
 			print("[PyNuclei] [WARN] Use stopAfter optional parameter only for templates, not for template categories")
 		
-		fileNameValidHost = f"{host.replace('/', FILE_SEPARATOR)}/"
+		if not isinstance(host, list):
+			host = [host]
+
+		currentTime = datetime.datetime.now()
+		timestampString = currentTime.strftime('%Y_%m_%d_%H_%M_%S')
+		hostNamesJoined = ','.join(host)
+
 		if not templates:
 			templates = self.nucleiTemplates
 
-		self.createResultDir(fileNameValidHost)
-
-		commands = list()
 		metricsPort = 9092
 		self.selectedTemplatesCount = len(templates)
 		
-		for template in templates:
```

Comment:
The multiple subprocess support for the list of templates and template category is removed here. It is there to support better concurrency and faster scan speed. 

Please restore the support for multiple subprocesses.

Reply:
Also, please update the ReadMe if there are any changes in function parameters and subprocess support